diff --git a/extensions/image_rotation/rotation-utils.c b/extensions/image_rotation/rotation-utils.c
index 06ce705f..c19412b3 100644
--- a/extensions/image_rotation/rotation-utils.c
+++ b/extensions/image_rotation/rotation-utils.c
@@ -227,7 +227,7 @@ _apply_transformation_async_thread (GTask        *task,
 	else {
 		// No changes
 		g_free (buffer);
-		g_task_return_boolean (task, true);
+		g_task_return_boolean (task, TRUE);
 		return;
 	}
 
@@ -245,7 +245,7 @@ _apply_transformation_async_thread (GTask        *task,
 		g_task_return_error (task, error);
 	}
 	else {
-		g_task_return_boolean (task, true);
+		g_task_return_boolean (task, TRUE);
 	}
 }
 
